packages/quantic/force-app/examples/main/lwc/exampleQuanticFacet/exampleQuanticFacet.js

```diff
@@ -2,9 +2,25 @@ import {api, LightningElement, track} from 'lwc';
 
 export default class ExampleQuanticFacet extends LightningElement {
     @api engineId = 'quantic-facet-engine';
-
-    isConfigured = false;
     @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Facet';
+    pageDescription = 'The Quantic Facet allows to refine search results by selecting one or more field values.'
```

Comment:
```suggestion
    pageDescription = 'The Quantic Facet allows you to refine search results by selecting one or more field values.'
```

Reply:
I think `...allows users to refine...` reads better. Thanks!